src/components/Book.js

```diff
@@ -1,28 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 
-function Book({ book, deleteBook }) {
+export default function Book({
+  id, title, author, category,
+}) {
+  const dispatch = useDispatch();
   return (
-    <div>
-      <p>
-        {book.title}
-        {' '}
-        by
-        {' '}
-        {book.author}
-      </p>
-      <button type="button" onClick={() => deleteBook(book.id)}>Delete</button>
-    </div>
+    <>
+      <div className="booklist">
+        <h2>List of Books</h2>
```

Comment:
[Optional] Please consider moving this title to your `Books.js` file so that it doesn't repeat whenever a new book is added👍

Reply:
Done. Thanks, again. :smiley: